index.html

```diff
@@ -1,417 +1,246 @@
+            <li class="list-item-text"><a href="#portfolio-section" class="link link-text">Portfolio</a></li>
+            <li class="list-item-text"><a href="#About" class="link link-text">About</a></li>
+            <li class="list-item-text"><a href="#contact-section" class="link link-text">Contact</a></li>
+            <li class="list-item-text"><a href="#" class="link link-text fas fa-envelope"></a></li>
+         </ul>
+        <div class="hamburger-menu" id="hamburger-menu"><i class="fas fa-bars"></i>
+      </div>
+    </nav>
+</header>
+    <section class="page-section first-section">
+    <h1 class="section-title title-l1">
+        Hey There. <br>I'm Ben
+    </h1>
+    <h3 class="section-subtitle subtitle-l1">
+        I'am a Software Developer
+    </h3>
+    <div class="intro-details">
+    <p class="description-text">
+        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don't hestiate to contact me.      
+    </p>   
+    <ul class="social-media-icons ul-list horizontal-list">
+        <li class="list-item-icon"><a href="https://www.github.com/telanet" class="link link-icon fa fa-github"></a></li>
+        <li class="list-item-icon"><a href="https://www.linkedin.com/in/benard-ngetich" class="link link-icon fa fa-linkedin"></a></li>
+        <li class="list-item-icon"><a href="#" class="link link-icon fa fa-angellist"></a></li>
+        <li class="list-item-icon"><a href="https://www.twitter.com/Ben54647605" class="link link-icon fa fa-twitter"></a></li>
+        <li class="list-item-icon"><a href="#" class="link link-icon fab fa-medium-m"></a></li>
+    </ul>
+    </div>
+    </section>
+    <section class="page-section works-section" id="portfolio-section">
+        <div class="box-join-title">
+            <h2 class="title section-title title-1">My Recent Works</h2>
+        <hr class="h-rule">
+        </div>
+          <div class="works-container">
+            <div class="posts-display">
+              <figure class="post-image">
+                <img src="images/img-placeholder-display.png" alt="Placeholder image">
+              </figure>
+              <div class="post-information">
+                <p class="post-title">Multi-Post Stories</p>
+                <p class="post-description description-text"> A daily selection of privately personalized reads; no accounts
+                  or sign-ups required. has been the industry's standard</p>
+                <ul class="post-tags ul-list horizontal-list">
+                  <li class="list-item-text">Html</li>
+                  <li class="list-item-text">Css</li>
+                  <li class="list-item-text">Bootstrap</li>
+                  <li class="list-item-text">Ruby</li>
+                </ul>
+                <button type="button" name="button" class="enable-button post-button">See project</button>
+              </div>
```

Comment:
- [ OPTIONAL] As required for this project your Design should match the Figma Design for this particular template and if we compare your design to Figma one they are not similar, kindly fix this and remove the big space between your cards for your project to look awesome. :wink: 
Your Design :point_down: 
![hero2](https://user-images.githubusercontent.com/88894525/193404163-6d1a3706-36e1-411b-9cbb-7c5dbc533638.png)

Figma design :point_down: 
![hero](https://user-images.githubusercontent.com/88894525/193404184-a93c89fc-3f7a-4b36-a1ca-a4d1cb46c121.png)

Reply:
Hi @Alfredbis29. Thanks for the feedback. All the changes you gave are optional, yet you did not approve the project. On the last optional change for the font-weight, when I use the Figma font weight it distort the whole form.  You could have assisted me better by recommending the changes I can use and the specific lines.